docs/self-hosted/install/install-on-kubernetes.md

```diff
@@ -4,45 +4,46 @@ url: /docs/self-hosted/latest/install/install-on-kubernetes/
 
 # Install Gitpod Self-Hosted on Kubernetes
 
-This section describes how to install Gitpod on any Kubernetes cluster.
-Gitpod also provides more optimized installations offering better performance for particular cloud providers:
-* *Google Cloud Platform*: Install Gitpod in a blank GCP project [using a script that automates the procedure](../install-on-gcp-script/).
-* *AWS*: Install Gitpod on AWS [using a script that automates the procedure](../install-on-aws-script/).
+This section describes how to install Gitpod on any Kubernetes cluster using [Helm](https://helm.sh). This ist the most flexible and generic way of installing Gitpod.  
 
-Gitpod is installed using [Helm](https://helm.sh).
-The source of the chart can be found [here](https://github.com/gitpod-io/gitpod/blob/master/chart/).
-Releases are published at https://charts.gitpod.io.
+For some cloud providers we offer [Terraform](https://www.terraform.io/) scripts packaged into an installer that set up the infrastructure around Gitpod following performance-relevant best practices and using the cloud provider's managed services: Kubernetes cluster, database, storage, image registry. We recommend using the installer if available for your cloud provider. Once the installer has created the cluster, it will pre-configure and install this Gitpod helm chart into it.
```

Comment:
1.st sentence is way too long. I skip it entirely during parsing.

Reply:
I've split into smaller parts.